setup.py

```diff
@@ -40,6 +40,9 @@ def get_install_requires() -> List[str]:
         "sqlalchemy>=1.1.0",
         "tqdm",
     ]
+    if sys.version_info[:2] > (3, 8):
+        requirements.append("Cython")
```

Comment:
Could you tell me why `Cython` is required for Python 3.9? Some packages are built during installation?

Reply:
Hope https://github.com/optuna/optuna/pull/1908/commits/481eafaf15b2e680ff0a4090335a59f848c2a26f satisfies you.

Reply:
Thanks!

Reply:
It'd help future maintainers if we know which libraries. I quickly tried this locally on my mac but didn't need Cython so it might be platform specific and coming sometime soon?